docs/quickstart/more_quickstart_options.md

```diff
@@ -40,38 +40,19 @@ Alternatively, you may opt to run the lakefs binary directly on your computer.
 
 1. Download the lakeFS binary for your operating system:
 
-   [Download lakefs](../index.md#downloads){: .btn .btn-green target="_blank"}
+   [Download lakefs](https://github.com/treeverse/lakeFS/releases){: .btn .btn-green target="_blank"}
 
-1. Create a configuration file:
-    
-   ```yaml
-   ---
-   database:
-     type: local
-     local:
-       path: "~/lakefs/metadata"
-    
-   blockstore: 
-     type: "local"
-     local:
-       path: "~/lakefs/data"
-   ```
-
-1. Create a local directories to store objects and metadata:
-
-   ```sh
-   mkdir -p ~/lakefs/data ~/lakefs/metadata
-   ```
-
-1. Run the server:
+2. Run the server:
     
    ```bash
-   ./lakefs --config /path/to/config.yaml run
+   ./lakefs run --local-settings
    ```
 
-1. Check your installation by opening [http://127.0.0.1:8000/setup](http://127.0.0.1:8000/setup){:target="_blank"} in your web browser.
+   It will create `~/lakefs` to store configuration and metadata. 
```

Comment:
You will find there `data` and `metadata` folders.
One for metadata used by lakefs and one will holds the data files uploaded though lakefs.
Maybe we can just specify it holds metadata and data.

Reply:
Fixed, thanks.